### DISCUSSION TITLE
Inches vs mm

### DISCUSSION THREAD

**Discussion by cuishuang413** [terjeio/ioSender#250] [Ideas]
Is there a way to display inches instead of mm?

Thanks!

**Comment by terjeio** [comment]
![image](https://user-images.githubusercontent.com/20260062/193982098-ba7b1848-ba2f-4c37-be0f-678ec87ef8fe.png)

**Reply by cuishuang413** [reply]
Thanks a lot for your reply. Still learning some basics here.
